src/LosslessWrappedERC20.sol

```diff
@@ -268,4 +268,24 @@ contract LosslessWrappedERC20 is
 
         return true;
     }
+
+    /// @notice This function cancels pending withdrawal request
+    function cancelWithdrawRequest() public {
+        Unwrapping storage unwrapping = unwrappingRequests[_msgSender()];
+
+        require(
+            unwrapping.unwrappingAmount > 0,
+            "LSS: No active withdrawal request"
+        );
+
+        require(
```

Comment:
I think we should allow cancelling even this requests. Cause if use change his mind, now he would need to withdraw and redeposit the funds again to get them secure.